hub/powertoys/grouppolicy.md

```diff
@@ -1,43 +1,124 @@
+4. If your domain has more than one domain controller, the new ADMX files will be replicated to them at the next domain replication interval.
 
 ### Scope
 
 You will find the policies under "Administrative Templates/Microsoft PowerToys" in both the Computer Configuration and User Configuration folders. If both settings are configured, the setting in Computer Configuration takes precedence over the setting in User Configuration.
 
 ## Policies
 
-Policy can force a PowerToys utility to be enabled or disabled for users covered by the policy.
+### Configure global utility enabled state
+
+This policy configures the enabled state for all PowerToys utilities.
+
+- If you enable this setting, all utilities will be always enabled and the user won't be able to disable it.
+- If you disable this setting, all utilities will be always disabled and the user won't be able to enable it.
+- If you don't configure this setting, users are able to disable or enable the utilities.
 
-### Configure enabled state
+The individual enabled state policies for the utilities will override this policy.
 
-For each utility shipped with PowerToys, there's a "Configure enabled state" policy, which forces an **enabled** state for the utility.
+### Configure enabled state for individual utilities
+
+For each utility shipped with PowerToys, there's a "Configure enabled state" policy, which forces and Enabled state for the utility.
 
 - If you enable this setting, the utility will be always enabled and the user won't be able to disable it.
 - If you disable this setting, the utility will be always disabled and the user won't be able to enable it.
-- If you don't configure this setting, users are able to disable or enable the utility at their own discretion.
+- If you don't configure this setting, users are able to disable or enable the utility.
+
+This policy has a higher priority than the policy "Configure global utility enabled state" and overrides it.
+
+### Allow experimentation
+
+This policy configures whether PowerToys experimentation is allowed. With experimentation allowed the user sees the new features being experimented if it gets selected as part of the test group. (Experimentation will only happen on Windows Insider builds.)
+
+- If this setting is not configured or enabled, the user can control experimentation in the PowerToys settings menu.
+- If this setting is disabled, experimentation is not allowed.
+- If this setting is not configured, experimentation is allowed.
+
+### Installer and Updates
+
+#### Disable per-user installation
+
+This policy configures whether PowerToys per-user installation is allowed or not.
+
+- If enabled, per-user installation is not allowed.
+- If disabled or not configured, per-user installation is allowed.
+
+ **Note:** You can set this policy only as Computer policy.
```

Comment:
The content on the page all looks good. Do you want to use the MS Learn Note style that stands out for the notes on this page?
```suggestion
> [!NOTE]
> You can set this policy only as Computer policy.
```